### PR TITLE
Event responsible group and group contact email, fixes #1104

### LINKED ISSUE #1104: Event responsible group field
Link events to group. This makes it easier for us to direct the end user to the actual responsible persons.

### PR DESCRIPTION
**Pull request by eirsyl**
Fixes #1104

### CODE REVIEW

File: lego/apps/events/models.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ class Event(Content, BasisModel, ObjectPermissionsModel):
     company = models.ForeignKey(
         Company, related_name='events', null=True, on_delete=models.SET_NULL
     )
+    responsible_group = models.ForeignKey(
+        'users.AbakusGroup', on_delete=models.CASCADE, null=True, related_name='events'
```
Comment (ekmartin) [inline lego/apps/events/models.py]: shouldn't this be `SET_NULL`?

File: lego/apps/events/views.py [hunk 2]
```diff
@@ -48,7 +48,9 @@ def get_queryset(self):
                 'pools', 'pools__registrations', 'company', 'tags'
             )
         elif self.action == 'retrieve':
-            queryset = Event.objects.prefetch_related('pools', 'pools__permission_groups', 'tags')
+            queryset = Event.objects.prefetch_related(
+                'pools', 'pools__permission_groups', 'tags', 'company', 'responsible_group'
```
Comment (ekmartin) [inline lego/apps/events/views.py]: shouldn't `responsible_group` be in a `select_related` since it's on the event model? and not a many to many field

Comment (eirsyl) [reply lego/apps/events/views.py]: Yeah my bad, to fast

File: lego/apps/events/serializers/events.py [hunk 3]
```diff
@@ -64,6 +65,9 @@ class EventReadDetailedSerializer(TagSerializerMixin, BasisModelSerializer):
     comment_target = CharField(read_only=True)
     cover = ImageField(required=False, options={'height': 500})
     company = CompanyField(queryset=Company.objects.all())
+    responsible_group = AbakusGroupField(
+        queryset=AbakusGroup.objects.all(), required=False, allow_null=True
```
Comment (ekmartin) [inline lego/apps/events/serializers/events.py]: think we should change this to `required=True` in the serializer eventually

**Comment by ekmartin** [comment]
good stuff

File: lego/apps/events/serializers/events.py (hunk 3 above)
Comment (eirsyl) [reply lego/apps/events/serializers/events.py]: yes, only required in the webapp for right now

**Pull request merged**